submission/loglist_refresher.go

```diff
@@ -30,9 +30,17 @@ const (
 	httpClientTimeout = 10 * time.Second
 )
 
+// LogListData wraps info on external LogList, keeping its JSON source and time
+// of download.
+type LogListData struct {
+	JSON         []byte
```

Comment:
Should this be a string?

Reply:
Good question: it originates from ReadFileOrURL at x509util. That func returns []byte.

Reply:
It makes sense that the util will return `[]byte` since it has no idea what's pointed at by the URI.
Mind you, it does look like the golang [stdlib json handling code](https://golang.org/pkg/encoding/json/#Unmarshal) deals in `[]byte` too, so maybe that's as good an answer as any.

Reply:
I'm leaving it as []byte for now. But we can return to this anytime later.